src/zeep/transports.py

```diff
@@ -5,16 +5,19 @@
 import requests
 
 from six.moves.urllib.parse import urlparse
+from zeep import __version__
 from zeep.cache import SqliteCache
-from zeep.utils import NotSet, get_version
 from zeep.wsdl.utils import etree_to_string
 
 
+ZEEP_USERAGENT = 'Zeep/%s (www.python-zeep.org)' % __version__
+
+
 class Transport(object):
     supports_async = False
 
-    def __init__(self, cache=NotSet, timeout=300, operation_timeout=None,
-                 verify=True, http_auth=None):
+    def __init__(self, cache=SqliteCache(), timeout=300, operation_timeout=None,
+                 verify=True, http_auth=None, http_headers={}):
```

Comment:
I would actually just go with using a memory cache by default. No dependency issues, and I feel like a library shouldn't write files to disk by default. If this is in fact the case.
What do you think?